docs/develop/dialog-api-in-office-add-ins.md

```diff
@@ -165,7 +165,7 @@ For an example of an add-in that does this, see the [Insert Excel charts using M
 
 ### Conditional messaging
 
-Because you can send multiple `messageParent` calls from the dialog box, but you have only one handler in the host page for the `DialogMessageReceived` event, the handler must use conditional logic to distinguish different messages. For example, if the dialog box prompts a user to sign in to an identity provider such as Microsoft Account or Google, it sends the user's profile as a message. If authentication fails, the dialog box sends error information to the host page, as in the following example:
+Because you can send multiple `messageParent` calls from the dialog box, but you have only one handler in the host page for the `DialogMessageReceived` event, the handler must use conditional logic to distinguish different messages. For example, if the dialog box prompts a user to sign in to an identity provider such as Microsoft account or Google, it sends the user's profile as a message. If authentication fails, the dialog box sends error information to the host page, as in the following example:
```

Comment:
suggestion: ...such as Microsoft or Google

Reply:
```suggestion
Because you can send multiple `messageParent` calls from the dialog box, but you have only one handler in the host page for the `DialogMessageReceived` event, the handler must use conditional logic to distinguish different messages. For example, if the dialog box prompts a user to sign in to an identity provider such as Microsoft or Google, it sends the user's profile as a message. If authentication fails, the dialog box sends error information to the host page, as in the following example:
```